analyses/molecular-subtyping-pathology/04-incorporate-pathology-feedback.Rmd

```diff
@@ -560,6 +566,55 @@ compiled_df <- compiled_df %>%
   distinct()
 ```
 
+## for HOPE cohort
+### 7316-1723
+```{r}
+
+compiled_df <- compiled_df %>% 
+  mutate(
+    integrated_diagnosis = case_when(
+      sample_id == "7316-1723" ~ "CIC-rearranged sarcoma",
+      sample_id == "7316-1746" ~ "Primary intracranial sarcoma, DICER1-mutant",
+      sample_id == "7316-194" ~ "Rosette-forming glioneuronal tumor",
+      sample_id == "7316-2151" ~ "Neuroepithelial tumor with PATZ1 fusion",
+      sample_id == "7316-2857" ~ "CNS tumor with BCOR internal tandem duplication",
+      sample_id == "7316-4844" ~ "Infantile hemispheric glioma, NTRK-altered", 
+      TRUE ~ integrated_diagnosis
+    ),
+    short_histology = case_when(
+      sample_id == "7316-1723" ~ "EWS",
+      sample_id == "7316-1746" ~ "EWS",
+      sample_id == "7316-194" ~ "LGAT",
+      sample_id == "7316-2151" ~ NA,
```

Comment:
I think we ought to keep 2151 with HGG broad/short histology/cancer group terms, per path review.